src/game/peers/ClientPeer.ts

```diff
@@ -32,11 +32,15 @@ export default class ClientPeer extends MessageEventEmitter<ClientMessage>
    * Mark peer as destroyed and clean up all listeners. Idempotent.
    */
   public destroy = (): void => {
-    this.destroyed = true
+    this._destroyed = true // eslint-disable-line no-underscore-dangle
     this.ext.removeAllListeners()
     this.removeAllListeners()
   }
 
+  public get destroyed(): boolean {
```

Comment:
This was to make debugging client peers easier without giving the outside world access to the `destroyed` variable.